### ISSUE TITLE
所有文件莫名全部被删除

### ISSUE THREAD

**Issue by zk199502** [microsoft/vscode#98094]
Issue Type: <b>Bug</b>

我下载了一个考拉的软化，后来在打开VSC后发现所有文件全部被删除，这是什么情况？回收站里也没有，整个电脑里也搜不到，还能恢复吗？两个月的心血啊！！

VS Code version: Code 1.45.1 (5763d909d5f12fe19f215cbfdd29a91c0fa9208a, 2020-05-14T08:27:35.169Z)
OS version: Windows_NT x64 10.0.17763

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i3-6100U CPU @ 2.30GHz (4 x 2304)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>viz_display_compositor: enabled_on<br>viz_hit_test_surface_layer: disabled_off_ok<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|3.87GB (1.00GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (7)</summary>

Extension|Author (truncated)|Version
---|---|---
open-html-in-browser|cod|0.1.21
vscode-view-in-browser|kop|1.0.5
vscode-autoprefixer|mrm|3.0.1
vscode-language-pack-zh-hans|MS-|1.45.1
debugger-for-chrome|msj|4.12.7
view-in-browser|qin|0.0.5
open-in-browser|tec|2.0.0

</details>

**Comment by AkiraVoid** [comment]
Translation Here: 
**All Files Were Deleted For No Reason**
Issue Type: **Bug**
I downloaded [Software's Name Here], then I opened my VSCode and found all files were deleted, why? The files were not in my Recycle Bin, and I also couldn't find them in my PC. Can them be recoveried? They are result of my 2 month's hardworking.
VS Code version: Code 1.45.1 (`[5763d90](https://github.com/microsoft/vscode/commit/5763d909d5f12fe19f215cbfdd29a91c0fa9208a)`, 2020-05-14T08:27:35.169Z)
OS version: Windows_NT x64 10.0.17763
